apps/meteor/server/services/omnichannel-integrations/providers/mobex.ts

```diff
@@ -1,11 +1,39 @@
 import { HTTP } from 'meteor/http';
 import { Base64 } from 'meteor/base64';
+import type { ISMSProvider, ServiceData, SMSProviderResult, SMSProviderResponse } from '@rocket.chat/core-typings';
```

Comment:
Not related to the code - can you make Omnichannel team the code owner for this new folder `omnichannel-integrations`

Reply:
Yup, but in a separate PR 🤗